src/libs/BetterLocation/Service/IpAddressV4Service.php

```diff
@@ -0,0 +1,46 @@
+<?php declare(strict_types=1);
+
+namespace App\BetterLocation\Service;
+
+use App\BetterLocation\BetterLocation;
+use App\BetterLocation\BetterLocationCollection;
+use App\Config;
+use App\MiniCurl\MiniCurl;
+
+final class IpAddressV4Service extends AbstractService
+{
+    const ID = 44;
+    const NAME = 'ipaddress';
```

Comment:
Since this merge request (MR) was created I already created another service and used ID 44. I just forgot, that this MR exists, sorry.

```suggestion
    public const ID = 45;
    public const NAME = 'IP address';
```